### PR TITLE
Docs 9546 edit pm config topics esb

### PR DESCRIPTION
**Pull request by emileemooreVT**

### CODE REVIEW

File: modules/ROOT/pages/configure-host.adoc [hunk 1]
```diff
@@ -1,21 +1,23 @@
 = Configure and Manage a Host Profile
```
Comment (vijayang) [inline modules/ROOT/pages/configure-host.adoc]: Since there is only one host org, please rephrase this as "Configure and Manage the Host Profile"

**Review by vijayang — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/configure-host.adoc (hunk 1 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-host.adoc]: Done!

**Review by vijayang — DISMISSED** [review]

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 2]
```diff
@@ -1,12 +1,14 @@
 = Create and Configure an Inbound Message Flow
 
-You can create and configure an inbound message flow either when you create a trading partner, or after you create a trading partner. 
+You can create and configure an inbound message flow either when you create a trading partner, or after you create a trading partner.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-message-flows.adoc]: "...either during creation or after you create a trading partner"?

Or no comma:
```suggestion
You can create and configure an inbound message flow either when you create a trading partner or after you create a trading partner.
```

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 
-After you create the inbound message flow, you configure the receive endpoint, source message type, target message type, and target endpoint. 
+Before you create your first message flow, you must configure your xref:setup-payload-storage.adoc[payload storage API].
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-message-flows.adoc]: inbound message flow?

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
+
+After you create the inbound message flow, you configure the receive endpoint, source message type, target message type, and target endpoint.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-message-flows.adoc]: But you can also perform this config. during creating the flow?

Comment (kbostrom) [reply modules/ROOT/pages/configure-message-flows.adoc]: So, something is missing here in the chronology or transition between prereqs:

This topic is Create and Configure a FLOW (Inbound).

The first sentence tells WHEN you can Create and Configure a FLOW: either during or after creating a TRADING PARTNER.
The second sentence is a prereq to CREATING a FLOW (and therefore also to creating a trading partner?).
The third sentence tells WHAT you can CONFIGURE AFTER you create the flow.

Comment (kbostrom) [reply modules/ROOT/pages/configure-message-flows.adoc]: So what I expect from the container topic (this Level-1 topic) is:
- Setting a context for the task
- Providing the answer to "Why?" the audience wants or needs to perform the task
- Setting audience expectations for the subtopics

Comment (kbostrom) [reply modules/ROOT/pages/configure-message-flows.adoc]: Are we saying that creating and configuring the flow are performed simultaneously, either during or after creating a partner?
Or are we saying that creating and configuring can be separate: that you can create first and configure later?
If you create first and configure later, at what point does creating a partner come in; what's the chronology?

Comment (kbostrom) [reply modules/ROOT/pages/configure-message-flows.adoc]: Perhaps this intro text is attempting to include too much of what belongs in subtopics? 
How about something like...:

Create and Configure an Inbound Message Flow

Before you can communicate with your trading partners, you must create an inbound message flow and then configure a receive endpoint, configure a message source type, import a translation map, and configure the target message type and endpoint.

Comment (kbostrom) [reply modules/ROOT/pages/configure-message-flows.adoc]: The second sentence should be a prereq. to the CREATE topic only.

Comment (kbostrom) [reply modules/ROOT/pages/configure-message-flows.adoc]: The third sentence introduces the configuration topics, but what is unclear to me from looking at them below how importing the translation map fits in: is that where the target type and endpoint are configured?

Comment (kbostrom) [reply modules/ROOT/pages/configure-message-flows.adoc]: END edit of this .adoc here, per request.

File: modules/ROOT/pages/Certificates.adoc [hunk 5]
```diff
@@ -1,58 +1,54 @@
 = Certificate Settings
 
-All certificates in Anypoint Partner Manager are securely stored in Anypoint Platform's secrets manager, which uses a secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.
+All certificates in Anypoint Partner Manager are securely stored in Anypoint Platform’s secrets manager, which uses secure vault technology to store and to control access to private keys, passwords, certificates, and other secrets.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]:
```suggestion
All certificates in Anypoint Partner Manager are securely stored in the Anypoint Platform secrets manager, which uses secure vault technology to store and to control access to private keys, passwords, certificates, and other secrets.
```
...to avoid making the (R) possessive

File: modules/ROOT/pages/Certificates.adoc [hunk 6]
```diff
@@ -1,58 +1,54 @@
...
 
-Configure your SSL certificate settings based on your certificate needs.
+You can configure your SSL certificate settings based on your certificate needs.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]: Per style ref., try to avoid modal verbs like "can," may," and "might" unless you are trying to distinguish between a step and an option.

Comment (kbostrom) [reply modules/ROOT/pages/Certificates.adoc]: Does one configure "settings"? Aren't settings themselves configuration?
And "configure your certificate based on your certificate needs" sounds kind of...unnecessary. What are we really trying to advise here?

File: modules/ROOT/pages/Certificates.adoc [hunk 7]
```diff
@@ -1,58 +1,54 @@
...
 
 The information provided for each certificate includes:
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]: Maybe this sentence can be reworded to replace the previous gratuitous statement:
```suggestion
Configure your certificate to include the following information:
```

File: modules/ROOT/pages/Certificates.adoc [hunk 8]
```diff
@@ -1,58 +1,54 @@
...
-Date from when the certificate is valid
-* *Expires* +
-Date after which the certificate is no longer valid 
-* *Endpoints* +
-Number of endpoints where the certificate is used
-* *Message Flows* +
-Number of message flows that use the certificate
+* *Authority*: The certificate provider
+* *Serial Number*: The serial number from the certificate
+* *Status*: The status of the certificate. The color coded icon on the certificate shows the user the status of the certificate:
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]:
```suggestion
* *Status*: The status of the certificate. 
+
The color-coded icon on the certificate shows the user the status of the certificate:
```

File: modules/ROOT/pages/Certificates.adoc [hunk 9]
```diff
@@ -1,58 +1,54 @@
...
+** Green: The certificate is active
+** Red: The certificate is inactive
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]: periods

File: modules/ROOT/pages/Certificates.adoc [hunk 10]
```diff
@@ -1,58 +1,54 @@
...
+* *Start Date*: The date from when the certificate is valid
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]:
```suggestion
* *Start Date*: The date from which the certificate is valid
```

File: modules/ROOT/pages/Certificates.adoc [hunk 11]
```diff
@@ -1,58 +1,54 @@
...
+* *Expires*: The date after which the certificate is no longer valid
+* *Endpoints*: The number of endpoints using the certificate
+* *Message Flows*: Number of message flows using the certificate
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]:
```suggestion
* *Message Flows*: The number of message flows using the certificate
```
...for parallelism with previous

File: modules/ROOT/pages/Certificates.adoc [hunk 12]
```diff
@@ -1,58 +1,54 @@
...
 
 In Partner Manager, you can:
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc:21]: This is a very vague introduction. Can we reword to somehow introduce the precise information following: certificate types and connection settings?

File: modules/ROOT/pages/Certificates.adoc [hunk 13]
```diff
@@ -1,58 +1,54 @@
...
 
 * Use a CA-signed certificate. +
-You do not need to upload an additional certificate because this endpoint already uses a CA-signed certificate. +
+If an endpoint is currently using a CA-signed certificate, you do not need to upload an additional certificate. +
 * Upload your own, self-signed certificate. +
-You can upload a new certificate (in `<certifcate name>.PEM` format), or view all the certificates available for use with this endpoint setup. 
-* Enable an insecure connection for local testing that doesn’t perform certificate validation by selecting *Allow insecure connection*. 
+You can upload a new certificate (in `<certifcate name>.PEM` format), or view all the certificates available for use with this endpoint setup.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]:
```suggestion
You can upload a new certificate (in `<certifcate name>.PEM` format) or view all the certificates available for use with this endpoint setup.
```

File: modules/ROOT/pages/Certificates.adoc [hunk 14]
```diff
@@ -1,58 +1,54 @@
...
+* Enable an insecure connection for local testing that doesn’t perform certificate validation by selecting *Allow insecure connection*.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc:27]: It's a bit difficult for me to parse the previous text and the new text, but it looks like this former 4-item list is now a 3-item list? Would you please check?

File: modules/ROOT/pages/Certificates.adoc [hunk 15]
```diff
@@ -1,58 +1,54 @@
...
 
-== Adding PEM Certificates
+== Add Certificates
 
 You can add certificates from the Host or Partner page.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]: Why are these page types init. capped?
If these are UI labels, then they require * *.

File: modules/ROOT/pages/Certificates.adoc [hunk 16]
```diff
@@ -1,58 +1,54 @@
...
 
-. In Partner Manager's left navigation, select *<host name>* or *Partners*.
-. In the *Certificates* section, click *New*. 
-. Click *Choose file* to upload a PEM certificate. 
-. Click *Upload Certificate*.
+. In Partner Manager's left navigation, select either *<host name>* or *Partners*. +
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]: Do we have to specify "Partner Manager's"? 
Is this just the home page, or does it have a label?

File: modules/ROOT/pages/Certificates.adoc [hunk 17]
```diff
@@ -1,58 +1,54 @@
...
+If you select Partners, select the parter to which you want to add the certificate from the list.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]:
```suggestion
If you select *Partners*, select from the list displayed the partner to which you want to add the certificate.
```

File: modules/ROOT/pages/Certificates.adoc [hunk 18]
```diff
@@ -1,58 +1,54 @@
...
+. In the *Certificates* section, click *New*.
+. In the *New certificate for <host name>* or the *New certificate for Partners* window, in the *Certificate* field, click *Choose file* to upload a certificate.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]:
```suggestion
. In the *Certificate* field of the *New certificate for <host name>* or the *New certificate for Partners* window, click *Choose file* to upload a certificate.
```

File: modules/ROOT/pages/Certificates.adoc [hunk 19]
```diff
@@ -1,58 +1,54 @@
...
+. Click *Upload Certificate*. +
+The new certificate is added to the host or the trading partner.
 
-== Deleting Certificates
+== Delete Certificates
 
-You can delete certificates that are not deployed or associated with endpoints. 
+You can delete certificates that are not deployed or associated with endpoints.
 
 You cannot delete a certificate if it is the only certificate that is uploaded.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]:
```suggestion
You can delete certificates that are not deployed or associated with endpoints; however, you cannot delete a certificate if it is the only certificate that is uploaded.
```

File: modules/ROOT/pages/Certificates.adoc [hunk 20]
```diff
@@ -1,58 +1,54 @@
...
 
 To delete a certificate:
 
-. In Partner Manager's left navigation, select *<host name>* or *Partners*.
+. In Partner Manager's left navigation, select *<host name>* or *Partners*. +
+If you select Partners, select the parter to which you want to add the certificate from the list.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]: ditto prev. edits to same text

File: modules/ROOT/pages/Certificates.adoc [hunk 21]
```diff
@@ -1,58 +1,54 @@
...
 . In the *Certificates* section, click the certificate to delete.
 . In the dialog window, click *Delete*.
```
Comment (kbostrom) [inline modules/ROOT/pages/Certificates.adoc]: dialog window? Isn't "a dialog" typically called "a dialog box"?

File: modules/ROOT/pages/configure-host.adoc [hunk 22]
```diff
@@ -1,21 +1,23 @@
-= Configure and Manage a Host Profile
+= Configure and Manage the Host Profile
 
-Your Anypoint Platform organization or your business group is referred to as the "host" in Anypoint Partner Manager. In Anypoint Platform, your host name is the same as your Anypoint Platform business group name. 
+Your Anypoint Platform organization or your business group is referred to as the "host" in Anypoint Partner Manager. In Anypoint Platform, your host name is the same as your Anypoint Platform business group name.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]: Use italic for introduction of terms, per SR:
```suggestion
Your Anypoint Platform organization or your business group is referred to as the _host_ in Anypoint Partner Manager. In Anypoint Platform, your host name is the same as your Anypoint Platform business group name.
```

File: modules/ROOT/pages/configure-host.adoc [hunk 23]
```diff
@@ -1,21 +1,23 @@
...
 
 Use the host page in Partner Manager to:
 
 * Add the EDI and AS2 xref:partner-manager-identifiers.adoc[identifiers] used by your organization.
-* Add digital certificates associated with your organization.
+* Add xref:certifates.adoc[digital certificates] associated with your organization.
 * Create and manage source and target xref:create-endpoint.adoc[endpoints].
 * Create and manage source and target xref:partner-manager-create-message-type.adoc[message types].
 * Configure details associated with the xref:setup-payload-storage-API.adoc[Payload Storage API].
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]: no periods

File: modules/ROOT/pages/configure-host.adoc [hunk 24]
```diff
@@ -1,21 +1,23 @@
...
- 
+
 
 
 == See Also
 
-* xref:partner-manager-identifiers.adoc[Sender and Receiver Identifiers] 
-* xref:delete-identifiers.adoc[Delete Identifiers] 
+* xref:access-management::organization.adoc[Organization]
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc:17]: This is a pretty vague title....

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 25]
```diff
@@ -16,47 +18,47 @@ Follow the process to xref:configure-partner.adoc#create-a-trading-partner[creat
...
 
 == Configure the Target Message Type
 
-. Expand the *Target at <your organization>* section. 
+. Expand the *Target at <your organization>* section.
 . Click *Select* to select the message type of your backend system to which to map the EDI transaction. +
-Partner Manager routes inbound EDI transactions based on matching identifiers configured at the Partner level. 
+Partner Manager routes inbound EDI transactions based on matching identifiers configured at the Partner level.
 
 == Configure the Target Endpoint
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-message-flows.adoc:62]: BREAK here to commit.

**Review by kbostrom — DISMISSED** [review]
First half of Docs 9546 CU, CE

File: modules/ROOT/pages/configure-message-flows.adoc (hunk 3 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-message-flows.adoc]: Done!

File: modules/ROOT/pages/Certificates.adoc (hunk 9 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/Certificates.adoc]: Done.

File: modules/ROOT/pages/Certificates.adoc (hunk 16 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/Certificates.adoc]: Done

File: modules/ROOT/pages/Certificates.adoc (hunk 20 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/Certificates.adoc]: Done

File: modules/ROOT/pages/Certificates.adoc (hunk 21 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/Certificates.adoc]: Done

File: modules/ROOT/pages/configure-host.adoc (hunk 23 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-host.adoc]: Done

File: modules/ROOT/pages/configure-message-flows.adoc (hunk 25 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-message-flows.adoc:62]: Got it!

File: modules/ROOT/pages/configure-host.adoc (hunk 24 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-host.adoc:17]: I do not own Access Management, so the title must stay.

File: modules/ROOT/pages/Certificates.adoc (hunk 6 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/Certificates.adoc]: Fixed - superfluous statement that was consolidated into the next line.

File: modules/ROOT/pages/Certificates.adoc (hunk 15 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/Certificates.adoc]: Once again, nameless pages. Changed to lower case.

File: modules/ROOT/pages/Certificates.adoc (hunk 14 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/Certificates.adoc:27]: There was one item that was removed because it was no longer valid and does not appear in the UI.

File: modules/ROOT/pages/Certificates.adoc (hunk 12 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/Certificates.adoc:21]: Done - added heading

File: modules/ROOT/pages/configure-partner.adoc [hunk 26]
```diff
@@ -1,6 +1,8 @@
 = Create, Configure, and Update Trading Partners
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc:1]: RESUME edit

Comment (kbostrom) [reply modules/ROOT/pages/configure-partner.adoc:1]: I realize that I have been commenting on text other than what you have recently changed, so I am limiting my edit of the remainder of the 3 PRs requested to the GREEN shaded, changed/new text.

File: modules/ROOT/pages/configure-partner.adoc [hunk 27]
```diff
@@ -1,6 +1,8 @@
...
 
-The Partner Manager *Partners* page lists all the configured trading partner profiles. 
+The Partner Manager *Partners* page lists all the configured trading partner profiles.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]: Can one "list" profiles? 
I would think you might list profile types or names, but rather display or otherwise show whole profiles.

File: modules/ROOT/pages/configure-partner.adoc [hunk 28]
```diff
@@ -1,6 +1,8 @@
...
+
+Before you create your first trading partner, you must configure your xref:setup-payload-storage.adoc[payload storage API].
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]: This is a prerequisite to CREATE, belonging in that task subtopic as a Prerequisite item or mentioned in the transition between subtitle and subtask Step 1. I gets lost in the intro to all three tasks, create, configure, and update.

File: modules/ROOT/pages/configure-partner.adoc [hunk 29]
```diff
@@ -24,13 +26,13 @@ To create a new partner, you need the following information:
...
-. Select an identifier from the *Type* list: 
+. Select an identifier from the *Type* list:
 * *AS2 Identity*
 * *Reference ID*
 * *X12-GS*
 * *X12-ISA*
-. In *Identifier Value*, enter a value to uniquely identify your partner. 
-. If you selected *X12-ISA* as the *Type*, you must also select a *Qualifier* from the list. 
+. In *Identifier Value*, enter a value to uniquely identify your partner.
+. If you selected *X12-ISA* as the *Type*, you must also select a *Qualifier* from the list.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]:
```suggestion
. If you selected *X12-ISA* for *Type*, you must also select a *Qualifier* value from the list.
```
OR
```suggestion
. If you selected *X12-ISA* as the type, you must also select a qualifier from the list.
```
Be careful to not treat labels as the noun objects themselves. The labels should not be preceded by an article unless they are also followed by a noun object: "the type," "the *Type* value," or "*Type," but not "the *Type*".
Make sense?

File: modules/ROOT/pages/configure-partner.adoc [hunk 30]
```diff
@@ -40,7 +42,7 @@ You can now xref:configure-message-flows.adoc[set up message flows] for this tra
 
 You can add and update the following information for each trading partner from the *Partners* tab in the left navigation:
 
-. In Partner Manager, select *Partners* in the left navigation menu. 
+. In Partner Manager, select *Partners* in the left navigation menu.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]: Intro and step kind of redundant

File: modules/ROOT/pages/configure-partner.adoc [hunk 31]
```diff
@@ -40,7 +42,7 @@ You can now xref:configure-message-flows.adoc[set up message flows] for this tra
...
 . Click the partner for whom to update the profile.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]:
```suggestion
. Click the partner whose profile you want to update.
```
oops...back to green....

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 32]
```diff
@@ -1,29 +1,33 @@
 = Payload Storage API for Partner Manager
```
Comment (kbostrom) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]: I'm not sure that we need this Level-1 Concept title; why not just start out with 
```suggestion
= Download and Deploy a Payload Storage API
```

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 33]
```diff
@@ -1,29 +1,33 @@
...
 
-Anypoint Partner Manager enables you to configure your own storage solution, such as database, Amazon S3, Azure Blob Storage, and so on to store transmission content, such as incoming and outgoing B2B message payloads.  
+Anypoint Partner Manager enables you to configure your own storage solution (for example, a database, Amazon S3, or Azure Blob Storage) to store transmission content, such as incoming and outgoing B2B message payloads.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]:
```suggestion
Anypoint Partner Manager enables you to configure your own storage solution (for example, a database, Amazon S3, or Azure Blob Storage) for transmission content, such as incoming and outgoing B2B message payloads.
```
"storage solution for transmission content" or other wording not as redundant as "storage solution to store transmission content" (a storage solution obviously stores...)

Comment (kbostrom) [reply modules/ROOT/pages/setup-payload-storage-API.adoc]: And do we need both "transmission content" and "incoming and outgoing B2B message payloads ("messages" would do) here?
Look for ways to reduce verbage in the legacy content: it is overinflated.

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 34]
```diff
@@ -1,29 +1,33 @@
...
 
-Before you create your first trading partner and message flow, you must configure your payload storage API within the host organization in Partner Manager.
+Before you create your first xref:configure-partner#create-partner[trading partner] and xref:configure-message-flow[message flow], you must configure your payload storage API within the host organization in Partner Manager.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]: This sentence alone could be the transition to the steps....

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 35]
```diff
@@ -1,29 +1,33 @@
...
 
-To store your content, you must implement an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage.
+To use your own storage solution for warehousing content, you must implement an API that provides Anypoint Partner Manager with a standardized way to connect to your storage.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]: "storage solution for warehousing content"?
And do we need to explain what an API is?

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 36]
```diff
@@ -1,29 +1,33 @@
...
 
-== Download and Deploy the Payload Storage API 
+== Download and Deploy the Payload Storage API
```
Comment (kbostrom) [inline modules/ROOT/pages/setup-payload-storage-API.adoc:7]: See above

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 37]
```diff
@@ -1,29 +1,33 @@
...
 
-Follow these steps to set up the payload storage API:
+=== Prerequisites
```
Comment (kbostrom) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]: If the title is DOWNLOAD and deploy, then this is not a prerequisite, right?

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 38]
```diff
@@ -1,29 +1,33 @@
...
 
-. Download these APIs from Anypoint Exchange:
-+
-https://www.mulesoft.com/exchange/com.mulesoft.b2b/partner-manager-content-storage-api[Partner Manager Content Storage API]
+Before you can deploy the Payload Storage API, you must first access and download the following APIs from Anypoint Exchange:
+
+. Access and download the following APIs from Anypoint Exchange:
```
Comment (kbostrom) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]: We don't need an intro and a step here, and there is only one step, anyway.
This is moot; this is not a prerequisite to downloading and deploying, because it *is* downloading.
Looks to me like this topic should be...:

==Download and Deploy...
intro
1. Download the following APIs .... (note plural; should the title etc. be plural "APIs"?)
2. Implement the API (APIs?)
3. Deploy the API, making note of the following values:
.
.
.
== Add the API (how is this different from deploying?)

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 39]
```diff
@@ -34,25 +38,18 @@ https://www.mulesoft.com/exchange/com.mulesoft.b2b/partner-manager-content-stora
...
 Your payload storage API must be accessible over the internet via HTTPS. Ensure that the appropriate authentication mechanisms are implemented to enable access over HTTPS.
 
-== Configure the Payload Storage API
+== Add the Payload Storage API
 
-. In Anypoint Platform, open Anypoint Partner Manager. 
-. In the navigation menu on the left, select your organization’s profile (host) page.
-. In the *Payload storage* section of the page, click *Edit* to add the path to your payload storage.  
+. In Anypoint Platform, open Anypoint Partner Manager.
+. In the navigation menu on the left, select your organization’s host page.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]: Note that previous tasks start with "In the Partner Manager left navigation menu,..."
We should standardize how all the tasks start, deciding whether we need to have the audience actually navigate to and open Partner Manager. 
Personally, I would start each task with "From the navigation menu [on the left], select...."

**Review by kbostrom — DISMISSED** [review]
Second half/complete CE, CU per request

File: modules/ROOT/pages/configure-partner.adoc (hunk 26 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-partner.adoc:1]: Gotcha!

File: modules/ROOT/pages/configure-message-flows.adoc (hunk 4 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-message-flows.adoc]: Updated into, moved prereq and deleted third sentence - does not provide any info that can't be discerned by reading the steps.

File: modules/ROOT/pages/configure-partner.adoc (hunk 27 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-partner.adoc]: Replaced "lists all the" with "displays".

File: modules/ROOT/pages/configure-partner.adoc (hunk 28 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-partner.adoc]: Moved to Create a Partner section

File: modules/ROOT/pages/configure-partner.adoc (hunk 30 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-partner.adoc]: Removed into

File: modules/ROOT/pages/setup-payload-storage-API.adoc (hunk 33 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/setup-payload-storage-API.adoc]: Removed "such as" section

File: modules/ROOT/pages/setup-payload-storage-API.adoc (hunk 36 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/setup-payload-storage-API.adoc:7]: MuleSoft users will know the definition of API, so that explanation is not required.

File: modules/ROOT/pages/setup-payload-storage-API.adoc (hunk 35 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/setup-payload-storage-API.adoc]: Changed to "to use your own storage solution" to match the intro paragraph. 


MuleSoft users will know the definition of API, so that explanation is not required.

File: modules/ROOT/pages/setup-payload-storage-API.adoc (hunk 34 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/setup-payload-storage-API.adoc]: Done

File: modules/ROOT/pages/setup-payload-storage-API.adoc (hunk 37 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/setup-payload-storage-API.adoc]: Removed!

File: modules/ROOT/pages/setup-payload-storage-API.adoc (hunk 38 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/setup-payload-storage-API.adoc]: Done!

Updated - ==Add the Payload Storage API to the Host Profile

File: modules/ROOT/pages/setup-payload-storage-API.adoc (hunk 39 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/setup-payload-storage-API.adoc]: Done!

**Review by kbostrom — APPROVED** [review]
Reapproval
